docs/Installation-Guide.rst

```diff
@@ -177,6 +179,151 @@ Docker
 
 Refer to `Docker folder <https://github.com/Microsoft/LightGBM/tree/master/docker>`__.
 
+Build Threadless Version (not Recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default build version of LightGBM is based on OpenMP.
+However, you can build the LightGBM without OpenMP support, but it is **strongly not recommended**.
+
+Windows
+^^^^^^^
+
+On Windows version of LightGBM without OpenMP support can be built using
+
+- **Visual Studio**;
+
+- **CMake** and **VS Build Tools**;
+
+- **CMake** and **MinGW**.
+
+Visual Studio (or VS Build Tools)
+*********************************
+
+With GUI
```

Comment:
Unfortunately, [I can't check](https://github.com/Microsoft/LightGBM/pull/1660#discussion_r216440029) whether it's enough just to uncheck OpenMP property. Can anyone test this, please?

Reply:
@StrikerRUS I just check it. It can work if merged https://github.com/Microsoft/LightGBM/tree/nomp-err

Reply:
create a new PR #1813 to fix that first

Reply:
@guolinke Great! Thanks!